templates/shared/secrets/add_list.tmpl

```diff
@@ -5,6 +5,10 @@
 			data-modal="#add-secret-modal"
 			data-modal-form.action="{{.Link}}"
 			data-modal-header="{{ctx.Locale.Tr "secrets.creation"}}"
+			data-modal-secret-name.value=""
+			data-modal-secret-name.read-only=""
```

Comment:
What's the `read-only`? Does it really work?

Reply:
Hmm ... it will assign `""` empty string to `el.readOnly`, which might break HTML DOM standard ....

Reply:
See my new commits.